docroot/profiles/custom/cgov_site/modules/custom/cgov_core/tests/src/Kernel/FieldStorage/CGovFieldStorageTest.php

```diff
@@ -143,6 +143,7 @@ public function testFields() {
     foreach ($this->fieldsToTest as $fieldToTest) {
       // Remove from our content type.
       $field = FieldConfig::loadByName('node', 'ponies', $fieldToTest["name"]);
+      /** @var \Drupal\Core\Entity\EntityInterface $field */
```

Comment:
this delete method is an instance of EntityInterface but I tried setting the instance of, still the error exists. so i have not removed @var as of now.

@bryanpizzillo

Reply:
Let's roll a local .patch to the core that fixes this type issue. (And any others we find.)

This way if the specific type changes in a 10.x branch we don't need to go fix it.